portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,38 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jazmine's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Jazmine's Portfolio</h1>
+      <p>Hello! My name is Jazmine Bullock.</p>
+      <img width = "340" src="images/pic-1.jpg" alt = "hey!"/>
+          <p>I am a Computer Science student at North Carolina A&T University.</p>
+          <p>I also am in the Blue and Gold marching machine! Go piccolos!</p>
+          <img width = "300" src="images/NCAT.png" alt = "school"/>
+          <img width = "300" src="images/group-pic.jpg" alt = "piccolos!"/>
+          <p>I was born in Elkton, MD, raised in Long Island, NY, and currently living in Dover, DE</p>
+          <img width = "250" src="images/Elkton.png" alt = "Long Island"/>
+          <img width = "250" src="images/long-island-ny-map.png" alt = "Elkton, MD"/>
+          <img width = "250" src="images/Dover.jpg" alt = "Dover, DE"/>
+          <p>Some of my favorite hobbies are reading, coding, and binge watching </p>
+          <p>I'm currently reading a book called Internment by Samira Ahmed. 10/10 I reccomend!</p>
+          <img width = "200" src="images/Itnerment.jpg" alt = "Internment book"/>
+          <p>I am also currently binge watching Avatar: The Last Air Bender. I am currently on the last season!!</p>
+          <img width = "200" src="images/TLA.jpg" alt = "Avatar: TLA"/>
+      <p>Click here to get a random quote from Avatar: The Last Air Bender:</p>
+      <button onclick="addRandomGreeting()">Click here!</button>
       <div id="greeting-container"></div>
+      <h2>My Links</h2>
+      <p><a href="https://www.linkedin.com/in/jbullockjazmine">LinkedIn</a></p>
```

Comment:
You can add target attributes to your anchor (a) tags to instruct links to open in new tabs/windows if you want.

`<a href="https://www.linkedin.com/in/jbullockjazmine" target="_blank">LinkedIn</a>`